pypeit/spectrographs/keck_deimos.py

```diff
@@ -278,7 +278,7 @@ def config_specific_par(self, scifile, inp_par=None):
         if self.get_meta_value(headarr, 'dispname') == '600ZD':
             par['calibrations']['wavelengths']['method'] = 'full_template'
             par['calibrations']['wavelengths']['reid_arxiv'] = 'keck_deimos_600ZD.fits'
```

Comment:
Two things. Shouldn't we be setting par['calibrations']['wavelengths']['lamps'] = 'use_header' somewhere here? Also, I'm a bit lost as to how the lamps can depend on what is written in the header, but not the reid_arxiv? Won't you get garbage results if lamps and reid_arxiv are not matched?